hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -227,6 +231,9 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6}>
             <OverviewCard loading={loading} title='Keys' data={keys.toString()} icon='file-text'/>
           </Col>
+          <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+            <OverviewCard loading={loading} title='Pending key Deletions' data={deletedKeys.toString()} icon='delete'/>
```

Comment:
nit

```suggestion
            <OverviewCard loading={loading} title='Pending Key Deletions' data={deletedKeys.toString()} icon='delete'/>
```

Reply:
Done Changes as Suggested by you from UI.